### PR TITLE
end of year update

### PR DESCRIPTION
**Pull request by windy54**
final update of the year

### CODE REVIEW

File: content/news/2024-december/_index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
++++
+title = 'December 2024'
+date = 0:0:0Z
```
Comment (danieljwarren) [inline content/news/2024-december/_index.md:3]:
```suggestion
date = 2024-12-18T00:00:00Z
```
Updated date to the expected format & date

File: content/news/2024-december/_index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+type = "posts"
+draft = true
```
Comment (danieljwarren) [inline content/news/2024-december/_index.md:5]:
```suggestion
```
Draft can be removed so the post will be seen when it goes live

File: content/news/2024-december/_index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
++++
+We have finished the year off with a trebuchet ecent, Christmas Crafts and the last PI Jam of the year.
```
Comment (danieljwarren) [inline content/news/2024-december/_index.md:7]:
```suggestion
We have finished the year off with a trebuchet event, Christmas Crafts and the last PI Jam of the year.
```
Spelling suggestion

File: content/news/2024-december/_index.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+Unfortunately due to ill health we only had two entries for the trebuchet contest.
+
+The Christmas Craft event was well attended with the Laser Cutter out to good use.
+
+Some visitors are already askking about next years event.
```
Comment (danieljwarren) [inline content/news/2024-december/_index.md:13]:
```suggestion
Some visitors are already asking about next years event.
```
Spelling suggestion

File: content/news/2024-december/_index.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+The last PI Jam was held on a Monday evening with the bew AI Camera on display.
+
+We now have quite a collection of PI5's and PICO's for members to use.
+
+Look out for more events in 2025.
+
+{{< groupimg width="420" >}}
```
Comment (danieljwarren) [inline content/news/2024-december/_index.md:21]:
```suggestion
{{< groupimg width="420" >}}

```
(Adding an empty line at the bottom to keep consistent with the style

**Review by danieljwarren — CHANGES_REQUESTED** [review]
@windy54 - I've made a couple of suggestions to the post. 
Each of them should then be possible to either commit directly, or add to a batch (and then commit)
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request

**Comment by windy54** [comment]
I was not sure if the time got updated automatically , so I will log in later on and accept the changes.CheersSteve GaleOn 19 Dec 2024, at 08:24, danieljwarren ***@***.***> wrote:﻿
@danieljwarren requested changes on this pull request.

@windy54 - I've made a couple of suggestions to the post.
Each of them should then be possible to either commit directly, or add to a batch (and then commit)
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request

In content/news/2024-december/_index.md:
> @@ -0,0 +1,21 @@
++++
+title = 'December 2024'
+date = 0:0:0Z

⬇️ Suggested change
-date = 0:0:0Z
+date = 2024-12-18T00:00:00Z


Updated date to the expected format & date



In content/news/2024-december/_index.md:
> @@ -0,0 +1,21 @@
++++
+title = 'December 2024'
+date = 0:0:0Z
+type = "posts"
+draft = true

⬇️ Suggested change
-draft = true


Draft can be removed so the post will be seen when it goes live



In content/news/2024-december/_index.md:
> @@ -0,0 +1,21 @@
++++
+title = 'December 2024'
+date = 0:0:0Z
+type = "posts"
+draft = true
++++
+We have finished the year off with a trebuchet ecent, Christmas Crafts and the last PI Jam of the year.

⬇️ Suggested change
-We have finished the year off with a trebuchet ecent, Christmas Crafts and the last PI Jam of the year.
+We have finished the year off with a trebuchet event, Christmas Crafts and the last PI Jam of the year.


Spelling suggestion



In content/news/2024-december/_index.md:
> @@ -0,0 +1,21 @@
++++
+title = 'December 2024'
+date = 0:0:0Z
+type = "posts"
+draft = true
++++
+We have finished the year off with a trebuchet ecent, Christmas Crafts and the last PI Jam of the year.
+
+Unfortunately due to ill health we only had two entries for the trebuchet contest.
+
+The Christmas Craft event was well attended with the Laser Cutter out to good use.
+
+Some visitors are already askking about next years event.

⬇️ Suggested change
-Some visitors are already askking about next years event.
+Some visitors are already asking about next years event.


Spelling suggestion



In content/news/2024-december/_index.md:
> ++++
+We have finished the year off with a trebuchet ecent, Christmas Crafts and the last PI Jam of the year.
+
+Unfortunately due to ill health we only had two entries for the trebuchet contest.
+
+The Christmas Craft event was well attended with the Laser Cutter out to good use.
+
+Some visitors are already askking about next years event.
+
+The last PI Jam was held on a Monday evening with the bew AI Camera on display.
+
+We now have quite a collection of PI5's and PICO's for members to use.
+
+Look out for more events in 2025.
+
+{{< groupimg width="420" >}}

⬇️ Suggested change
-{{< groupimg width="420" >}}
+{{< groupimg width="420" >}}


(Adding an empty line at the bottom to keep consistent with the style

**Comment by windy54** [comment]
i have made the edits and pushed to december24 branch

Steve

**Review by danieljwarren — APPROVED** [review]
Changes look good!

**Review by jtljac — APPROVED** [review]

**Pull request merged**